.github/workflows/staging-smoke-tests.yaml

```diff
@@ -12,6 +12,16 @@ jobs:
     steps:
       - name: "Checkout jade-data-repo"
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: "Get previous tag"
+        id: apiprevioustag
+        uses: "broadinstitute/github-action-get-previous-tag@master"
```

Comment:
Can you explain to me what this is doing? I read the git action docs, it says that this action will return the latest tag on the repo (the name is confusing!). Is the latest tag typically not the same as HEAD on master?

Reply:
We checkout the repository, find the latest tag (== semantic version, different from the git hash), and then switch to the repository at that state. The semantic version could lag the dev HEAD branch by a few commits, although this should be minimized now that every PR bumps and deploys the code.

Reply:
I'm still unclear as to why it's done this way. If a tag is generated on PR merge to develop, the only time this isn't equivalent to checking out develop is when there's a release from master that's been tagged.